components/function-controller/internal/controllers/serverless/function_reconcile_gitops_test.go

```diff
@@ -803,14 +815,18 @@ func Test_stateFnGitCheckSources(t *testing.T) {
 			},
 		}
 
-		m := new(automock.GitOperator)
-		m.On("LastCommit", mock.Anything).Return("", fmt.Errorf("test error")).Once()
+		gitClient := new(automock.GitClient)
+		gitClient.On("LastCommit", mock.Anything).Return("", fmt.Errorf("test error")).Once()
```

Comment:
We have `Once` constraint in this place, but I can't see related `AssertExpectations`.